R/Spectra-functions.R

```diff
@@ -71,6 +129,9 @@ addProcessing <- function(object, FUN, ...) {
 #'     matrix will be passed to the first argument of the function. The function
 #'     should also have arguments `...`.
 #'
+#' @param spectraVariables `character` defining spectra variables that should
```

Comment:
Aren't there default parameters that are always passed? Or is that not the case anymore?

Reply:
now, by default, we're only passing the peak matrix. I would keep it to the user/developer that he/she defines which variables he/she needs in the function. that is also slightly faster and also simplifies the function to be applied to the peaks, because there is no longer the need that certain parameters (`spectrumMsLevel` and `centroided`) have to be present.